src/olympia/files/models.py

```diff
@@ -727,6 +732,23 @@ class Meta:
         db_table = 'webext_permissions'
 
 
+Permission = namedtuple('Permission',
+                        'name, description')
+
+
+class WebextPermissionDescription(ModelBase):
+    MATCH_ALL_REGEX = r'^\<all_urls\>|(\*|http|https):\/\/\*\/\*'
+    ALL_URLS_PERMISSION = Permission(
+        u'all_urls',
+        _lazy(u'Access your data for all websites')
+    )
+    name = models.CharField(max_length=255)
```

Comment:
`name` should be `unique=True` (and corresponding db constraint added to the migration), right ?